### PR TITLE
Codefix that makes class internal fix #152

### LINKED ISSUE #152: Codefix that makes class internal

### PR DESCRIPTION
**Pull request by andrewtechnology**

### CODE REVIEW

File: Gu.Analyzers.CodeFixes/Gu.Analyzers.CodeFixes.csproj [hunk 1]
```diff
@@ -18,5 +18,11 @@
   <ItemGroup>
     <ProjectReference Include="..\Gu.Analyzers.Analyzers\Gu.Analyzers.Analyzers.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="MakeInternalFixProvider.cs">
```
Comment (JohanLarsson) [inline Gu.Analyzers.CodeFixes/Gu.Analyzers.CodeFixes.csproj]: This looks weird, is it needed for anything?

File: Gu.Analyzers.CodeFixes/MakeInternalFixProvider.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            foreach (var modifier in typeDeclaration.Modifiers)
+            {
+                if (modifier.Kind() == SyntaxKind.PublicKeyword)
+                {
+                    var syntaxToken = SyntaxFactory.Token(SyntaxKind.InternalKeyword);
+                    return Task.FromResult(document.WithSyntaxRoot(root.ReplaceToken(modifier, syntaxToken)));
+                }
+            }
+
+            return null;
```
Comment (JohanLarsson) [inline Gu.Analyzers.CodeFixes/MakeInternalFixProvider.cs]: We probably want to return `Task.FromResult(document)  here

File: Gu.Analyzers.CodeFixes/Gu.Analyzers.CodeFixes.csproj (hunk 1 above)
Comment (andrewtechnology) [reply Gu.Analyzers.CodeFixes/Gu.Analyzers.CodeFixes.csproj]: Nope. I think my visual studio just added it for no reason.

File: Gu.Analyzers.CodeFixes/MakeInternalFixProvider.cs (hunk 2 above)
Comment (andrewtechnology) [reply Gu.Analyzers.CodeFixes/MakeInternalFixProvider.cs]: Ok.

**Pull request merged**